### PR TITLE
Add featured project

### PR DESCRIPTION
**Pull request by laurenzfg**
Include Vacstats in featured projects

### CODE REVIEW

**Review by Rubber1Duck — CHANGES_REQUESTED** [review]
You have to add your project to /docs/index.md, too!

File: docs/index.md [hunk 1]
```diff
@@ -148,6 +148,7 @@ Now you can access the server at `http://localhost:8080`.
 - Wordpress Widget zur Anzeige der letzen 3 Inzidenzen für einen Landkreis [https://github.com/p-tenz/widget-7d-incidence](https://github.com/p-tenz/widget-7d-incidence)
 - Informatives Dashboard, zum schnellen Überblick des Status der Corona-Schutzimpfungen in Deutschland [https://www.impfung.io/](https://www.impfung.io/)
 - Discordbot at [https://germanycovid.de/discord](https://germanycovid.de/discord)
+- Web App for vaccination numbers which issues a Push Notification whenever the new vaccination numbers are available (https://vacstats.laurenzfg.com)
```
Comment (Rubber1Duck) [inline docs/index.md]: please add "[https://vacstats.laurenzfg.com]" bevor "(https://vacstats.laurenzfg.com)"

**Review by Rubber1Duck — CHANGES_REQUESTED** [review]

File: docs/index.md (hunk 1 above)
Comment (laurenzfg) [reply docs/index.md]: sure! sorry for not complying to markdown in my PR.

**Review by Rubber1Duck — APPROVED** [review]

**Comment by Rubber1Duck** [comment]
No that looks fine to me! @marlon360 has to merge this!

**Comment by laurenzfg** [comment]
hey @marlon360
I am currently reviewing my open PRs and I would love to either close or have this PR merged. Do you have capacity to merge this?

On an unrelated side note: I will change the API host of my project to your public node, since I don't wamt to keep my fork up forever.

**Pull request merged**